frontend/src/pages/DetailedCourse/DetailedCourseContainer.tsx

```diff
@@ -4,7 +4,7 @@ import { useNavigate, useParams } from 'react-router';
 import { useApplyCourse, useGetCourseInfo } from 'api/courses';
 import { useGetClientCourseInfo } from 'api/myCourses';
 import { useGetProfile } from 'api/profile';
-import useGetMyCourses from 'api/myCourses/getClientCourses';
+import { useGetMyCourses } from 'api/myCourses/getClientCourses';
```

Comment:
it's already exported from api/myCourses/index.ts, so you can import it like this: `import { useGetMyCourses } from 'api/myCourses';`